docs/extensions/hooks.md

```diff
@@ -81,6 +81,13 @@ The context object has the following properties:
 - `schema` — The current API schema in use
 - `accountability` — Information about the current user
 
+::: warning Breaking app/admin
+
+By modifying the expected read output, especially for system collections. It is very easy to accidentally brick the app
+or api response.
+
+:::
+
```

Comment:
This doesn't feel quite right, and I don't think it's placed correctly. 

I would suggest something with more explanation: 

> Directus reads system collection data to perform correctly, both in the Data Studio and the generated APIs. Be careful when modifying the output of system collection read/query events, as this can cause Directus core functionality to break. 

And them move it to the bottom of the system collection tips for tables you are modifying with this PR.